plugins/defaultplugins/aclplugin/vppdump/dump_vppcalls.go

```diff
@@ -173,22 +432,20 @@ func getIPACLDetails(vppChannel *govppapi.Channel, idx uint32) (*acl.AccessLists
 
 		rule.Matches = &matches
 		rule.Actions = &actions
-		rules = append(rules, &rule)
+		ruleData = append(ruleData, &rule)
 	}
 
-	return &acl.AccessLists_Acl{Rules: rules, AclName: string(bytes.Trim(reply.Tag, "\x00"))}, nil
+	return &acl.AccessLists_Acl{Rules: ruleData, AclName: string(bytes.Trim(reply.Tag, "\x00"))}, nil
 }
 
-// getIPRule translates an IP rule from the binary VPP API format into the
+// getIPRuleMatches translates an IP rule from the binary VPP API format into the
 // ACL Plugin's NB format
-func getIPRule(r acl_api.ACLRule) *acl.AccessLists_Acl_Rule_Matches_IpRule {
+func getIPRuleMatches(r acl_api.ACLRule) *acl.AccessLists_Acl_Rule_Matches_IpRule {
 	ipRule := acl.AccessLists_Acl_Rule_Matches_IpRule{}
 
-	saddr := net.IPNet{IP: r.SrcIPAddr, Mask: []byte{}}
-	daddr := net.IPNet{IP: r.DstIPAddr, Mask: []byte{}}
 	ip := acl.AccessLists_Acl_Rule_Matches_IpRule_Ip{
-		SourceNetwork:      fmt.Sprintf("%s/%d", saddr.String(), r.SrcIPPrefixLen),
-		DestinationNetwork: fmt.Sprintf("%s/%d", daddr.String(), r.DstIPPrefixLen),
+		SourceNetwork:      fmt.Sprintf("%v/%d", decodeIPv4Address(r.SrcIPAddr), r.SrcIPPrefixLen),
+		DestinationNetwork: fmt.Sprintf("%s/%d", decodeIPv4Address(r.DstIPAddr), r.DstIPPrefixLen),
```

Comment:
Formatting seems inconsistent here. Probably makes no difference in output since it returns string, but it should be same for both cases. Either use `%s` or `%v` in both cases.